task_02/ex7.py

```diff
@@ -0,0 +1,24 @@
+"""Index Power."""
+
+# https://py.checkio.org/en/mission/index-power/
+
+
+def index_power(list_arg, n):
+    """You are given an array with positive numbers and a number N.
+
+    You should find the Nth power of the element in the array with the index N.
+    If N is outside of the array, then return -1.
+    """
+    if type(list_arg) is not list:
+        raise Exception("Invalid argument type")
```

Comment:
The same comment why can't it be a tuple? And I really don't understand what should I type to input to catch exactly this exception?